### DISCUSSION TITLE
Make removed comments red

### DISCUSSION THREAD

**Discussion by iTrooz** [Docile-Alligator/Infinity-For-Reddit#980] [Ideas]
Small improvement idea : Make the removed comments red (like, the text color) when you use the "see removed comment" button to see them

I'm willing to submit a PR for this, if you are okay with the feature

(By the way, I think this feature should be shown in the README, I didn't know it was here until today)

**Comment by MysteriousOrb2000** [comment]
This would definitely help with usability.

**Comment by iTrooz** [comment]
Yeah I think so too !
@Docile-Alligator Would you like a PR ?

**Comment by Wladefant** [comment]
Here you can also add a [Feature Request] to the title 😀

**Comment by Docile-Alligator** [comment]
@iTrooz Thank you! That's brilliant!

**Comment by ashpakouski** [comment]
@iTrooz, are you still going to submit pull request?

**Comment by iTrooz** [comment]
> @iTrooz, are you still going to submit pull request?

Yep, trying to work on it
It's my first time doing Android development tho so it may take a long time, so feel free to do it yourself if this was your intention
